roles/ovirt-repositories/README.md

```diff
@@ -0,0 +1,69 @@
+
+oVirt Repositories
+==================
+
+The `oVirt.repositories` role is used to set the repositories required for oVirt engine installation.
```

Comment:
"... for oVirt engine or host installation."

Hopefully later we will be able to use that role also to setup required repos on hosts ...

Reply:
Fixed.